Realm/Realm/RealmConfigurationBase.cs

```diff
@@ -118,9 +118,12 @@ public byte[] EncryptionKey
             }
         }
 
-        internal RealmConfigurationBase(string optionalPath)
+        internal RealmConfigurationBase(string optionalPath = null, bool calculatePath = true)
```

Comment:
Maybe we should just add a default empty constructor instead?

Reply:
Much better, yes. Wonder why it didn't occur to me 😔